dashbuilder/dashbuilder-backend/dashbuilder-dataset-cdi/src/test/java/BootstrapTest.java

```diff
@@ -37,6 +38,7 @@
 import static org.mockito.Mockito.*;
 
 @RunWith(Arquillian.class)
+@Ignore // see https://issues.jboss.org/browse/RHPAM-832
```

Comment:
Does it make sense to keep these tests if they're never going to be executed anyway? If there's no work planned for unignoring these we might as well delete these, because nobody will ever fix these.

If you insist on keeping the tests and just having those ignored I'd suggest:
1. add reason for ignoring as `value` of the annotation (to make it accessible to tooling) instead of just plain comment. I.e. use `@Ignore{"bla"}` instead of `@Ignore //bla`
2. Add either inline comment or info to the jira stating when these could be unignored or what further steps / investigations need to be done to make it possible to unignore these.

Reply:
Done 1 & 2.